package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "analetica-backend",
+  "version": "1.0.0",
+  "description": "backend of the webiste",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "node src/index.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/sushantbasak/Analetica-Backend.git#analetica-backend"
+  },
+  "author": "Akhil",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/sushantbasak/Analetica-Backend/issues"
+  },
+  "homepage": "https://github.com/sushantbasak/Analetica-Backend/tree/analetica-backend#readme",
+  "dependencies": {
+    "express": "^4.17.1",
+    "nodemon": "^2.0.6"
```

Comment:
We should add `nodemon` as a dev dependency, as you don't need `nodemon` to run it in production.

You can first remove `nodemon` as a dependency and then add it again as a dev dependency by running `npm i --save-dev nodemon`